challenges/qr_code_component/index.html

```diff
@@ -28,35 +28,56 @@
     <link rel="manifest" href="../site.webmanifest" />
     <link rel="stylesheet" href="style.css" />
   </head>
-  <body class="m-0">
+  <body class="font-outfit m-0">
     <div
-      class="flex h-[100vh] w-full flex-col place-content-center text-center"
+      class="bg-lightGray flex min-h-screen flex-col items-center justify-center"
     >
-      <div>
-        <a
-          class="text-optimum-blue hover:text-optimum-darkblue"
-          href="https://www.frontendmentor.io/challenges/qr-code-component-iux_sIO_H"
-          target="_blank"
-          >Frontend Mentor challenge</a
-        >
-      </div>
-      <div>
-        for
-        <a
-          class="text-optimum-blue hover:text-optimum-darkblue"
-          href="https://github.com/optimumBA"
-          target="_blank"
-          >Optimum BH</a
-        >
-        Internship.
+      <div class="w-80 overflow-hidden rounded-lg border-2 bg-white p-5">
+        <img
+          class="mx-auto rounded-lg"
+          src="images/image-qr-code.png"
+          alt="qr code"
+        />
+        <div class="px-5 text-center">
+          <h2 class="text-darkBlue my-5 text-lg font-bold">
+            Improve your front-end skills by building projects
+          </h2>
+          <p class="text-grayishBlue pb-5 text-sm">
+            Scan the QR code to visit Frontend Mentor and take your coding
+            skills to the next level
+          </p>
+        </div>
       </div>
 
-      <div class="mt-3">
-        <a
-          class="text-optimum-blue hover:text-optimum-darkblue"
-          href="https://github.com/optimumBA/frontend_mentor_challenges/tree/qr_code_component/challenges/qr_code_component"
-          >Source code</a
-        >
+      <div
+        class="absolute top-0 z-30 flex w-full place-content-center text-center text-xs"
+      >
+        <div>
+          <a
+            class="text-optimum-blue hover:text-optimum-darkblue"
+            href="https://www.frontendmentor.io/challenges/qr-code-component-iux_sIO_H"
+            target="_blank"
+            >Frontend Mentor challenge</a
+          >
+        </div>
+        <div>
+          for
+          <a
+            class="text-optimum-blue hover:text-optimum-darkblue"
+            href="https://github.com/optimumBA"
+            target="_blank"
+            >Optimum BH</a
+          >
+          Internship.
+        </div>
+
+        <div>
+          <a
+            class="text-optimum-blue hover:text-optimum-darkblue"
+            href="https://github.com/optimumBA/frontend_mentor_challenges/tree/qr_code_component/challenges/qr_code_component"
```

Comment:
Put your username instead of `optimumBA` so that visitors can get to your code when they click on the "Source code" link.

Reply:
Thanks for the comments. Changes have been made. Check my latest commit

Reply:
Are you saying that I replace "Source Code" with my GitHub username, or just the url and leave "Source Code" text as is?

Reply:
You've done it correctly.